modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/VirtualFlowTest.java

```diff
@@ -1523,6 +1525,117 @@ protected double computeMaxHeight(double width) {
         assertEquals(3, firstCell.getIndex());
         assertEquals(-10, firstCell.getLayoutY(),1);
     }
+
+    /**
+     * The VirtualFlow should never call the compute height methods when a fixed cell size is set.
+     */
+    @Test
+    public void testComputeHeightShouldNotBeCalledWhenFixedCellSizeIsSet() {
+        int cellSize = 24;
+
+        flow = new VirtualFlowShim<>();
+        flow.setFixedCellSize(cellSize);
+        flow.setCellFactory(p -> new CellStub(flow) {
+
+            @Override
+            protected double computeMinHeight(double width) {
+                return 1337;
```

Comment:
Can we add a final variable for this number since it is repeated many times?

Reply:
yeah, I don't think it's that important.